stream-compositions/services/legal-entity-composition-service/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,29 @@
+spring:
+  application:
+    name: legal-entity-composition-service
+  cloud:
```

Comment:
thinking about other projects that might be using Kubernetes as infrastructure here, maybe we shouldn't hardcode eureka and spring config server in the bootstrap.yaml, I would suggest maybe creating a separate profile for that

Reply:
Updated.